Glaxnimate build error

**Describe the bug**
A clear and concise description of what the bug is.

The qt-6 version is 6.11.0, and Glaxnimate encounters a build error on the new version of MLT, v7.38.0
The following two options have been turned on:
` '-DMOD_QT6=ON' `and `'-DMOD_GLAXNIMATE_QT6=ON'`
#1217 It has caused the failure of my new version of mlt build. I hope it can be fixed
```
CMake Error at src/modules/glaxnimate/glaxnimate/src/core/CMakeLists.txt:144 (target_link_libraries):
  Target "GlaxnimateCoreBase" links to:

    Qt5::Core

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

```

**To Reproduce**
Steps to reproduce the behavior:
Build log.
```
[INFO]:  Build template selected: cmakeninja
[INFO]:  cmakeninja > Running configure step ...
[INFO]:  Creating directory for shadow build ...
mkdir: created directory '/var/cache/acbs/build/acbs.x5p75or_/mlt/abbuild'
[INFO]:  Running CMakeLists.txt to generate Ninja Configuration ...
-- The C compiler identification is GNU 15.2.0
-- The CXX compiler identification is GNU 15.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found PkgConfig: /usr/bin/pkg-config (found version "2.5.1")
-- Checking for module 'libexif'
--   Found libexif, version 0.6.26
-- Checking for module 'fontconfig'
--   Found fontconfig, version 2.17.1
-- Found FFmpeg: /usr/lib/libavformat.so;/usr/lib/libswscale.so;/usr/lib/libavutil.so;/usr/lib/libavcodec.so;/usr/lib/libavfilter.so;/usr/lib/libavdevice.so;/usr/lib/libswresample.so  found components: AVFORMAT SWSCALE AVUTIL AVCODEC AVFILTER AVDEVICE SWRESAMPLE
-- Checking for module 'frei0r'
--   Found frei0r, version 2.5.1
-- Checking for module 'gdk-pixbuf-2.0'
--   Found gdk-pixbuf-2.0, version 2.44.4
-- Checking for module 'pango'
--   Found pango, version 1.56.1
-- Checking for module 'pangoft2'
--   Found pangoft2, version 1.56.1
-- Found JACK: /usr/lib/libjack.so (found version "1.9.22")
-- Checking for module 'glib-2.0'
--   Found glib-2.0, version 2.86.3
-- Checking for module 'libxml-2.0'
--   Found libxml-2.0, version 2.10.4
-- Looking for ladspa.h
-- Looking for ladspa.h - found
-- Checking for module 'lilv-0'
--   Found lilv-0, version 0.24.12
-- Checking for module 'movit'
--   Found movit, version git
-- Found OpenGL: /usr/lib/libOpenGL.so
-- Found X11: /usr/include
-- Looking for XOpenDisplay in /usr/lib/libX11.so;/usr/lib/libXext.so
-- Looking for XOpenDisplay in /usr/lib/libX11.so;/usr/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found OpenCV: /usr (found version "4.13.0") found components: tracking
-- Checking for module 'libebur128'
--   Found libebur128, version 1.2.6
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /usr/include
-- Checking for module 'samplerate'
--   Found samplerate, version 0.1.9
-- Checking for module 'rtaudio'
--   Package 'rtaudio' not found
-- Checking for module 'alsa'
--   Found alsa, version 1.2.15.3
-- Checking for module 'libpulse-simple'
--   Found libpulse-simple, version 17.0
-- Checking for module 'rubberband'
--   Found rubberband, version 4.0.0
-- Checking for module 'sdl'
--   Found sdl, version 1.2.72
-- Checking for module 'sox'
--   Found sox, version 14.4.2
-- Checking for module 'vidstab'
--   Found vidstab, version 0.98
-- Checking for module 'vorbis'
--   Found vorbis, version 1.3.7
-- Checking for module 'vorbisfile'
--   Found vorbisfile, version 1.3.7
-- Checking for module 'libxml-2.0'
--   Found libxml-2.0, version 2.10.4
-- Found SWIG: /usr/bin/swig (found version "4.4.1")
-- Found JNI: /usr/lib/java/include  found components: AWT JVM
-- Found Lua: /usr/lib/liblua5.1.so;/usr/lib/libm.so (found version "5.1.5")
-- Found Perl: /usr/bin/perl (found version "5.36.3")
-- Found PerlLibs: /usr/lib/perl5/core_perl/CORE/libperl.so (found version "5.36.3")
-- Found PHP: /usr/bin/php-config (found version "8.5.4")
-- Found Python3: /usr/bin/python3.14 (found version "3.14.4") found components: Interpreter Development Development.Module Development.Embed
-- Found Ruby: /usr/bin/ruby (found suitable version "3.4.7", minimum required is "1.8.0")
-- Found Tclsh: /bin/tclsh (found version "8.6")
-- Found TCL: /usr/lib/libtcl.so
-- Found TCLTK: /usr/lib/libtcl.so
-- Found TK: /usr/lib/libtk.so
-- Could NOT find ClangFormat (missing: CLANGFORMAT_EXECUTABLE CLANGFORMAT_VERSION) (Required is exact version "14")
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Building Glaxnimate core without KDE
-- Found OpenMP_C: -fopenmp (found version "4.5")
-- Found OpenMP_CXX: -fopenmp (found version "4.5")
-- Found OpenMP: TRUE (found version "4.5")
-- Found ZLIB: /usr/lib/libz.so (found version "1.3.1")
-- Glaxnimate Modules Enabled:
--     ThorVG
--     Gzip
--     ExtraFormats
-- The following features have been enabled:

 * GPLv2
 * GPLv3
 * Module: avformat
 * Module: DeckLink
 * Module: Frei0r
 * Module: GDK
 * Module: Glaxnimate (Qt6)
 * Module: JACKRack
 * Module: Kdenlive
 * Module: Movit
 * Module: Normalize
 * Module: Oldfilm
 * Module: OpenCV
 * Module: OpenFX
 * Module: Plus
 * Module: PlusGPL
 * Module: Qt6
 * Module: Resample
 * Module: RtAudio
 * Module: Rubberband
 * Module: SDL1
 * Module: SDL2
 * Module: SoX
 * Module: vid.stab
 * Module: Vorbis
 * Module: XINE
 * Module: XML
 * SWIG: Java
 * SWIG: Lua
 * SWIG: Perl
 * SWIG: PHP
 * SWIG: Python
 * SWIG: Ruby
 * SWIG: Tcl
 * lv2: LV2 Plugins support
 * vst2: VST2 Plugins support

-- The following OPTIONAL packages have been found:

 * FFTW3
 * JACK
 * Freetype
 * Fontconfig
 * Qt6Core
 * Vulkan
 * WrapVulkanHeaders
 * Qt6DBusTools (required version >= 6.11.0)
 * Qt6Gui
 * Qt6Xml
 * Qt6CoreTools (required version >= 6.11.0)
 * Qt6GuiTools (required version >= 6.11.0)
 * Qt6WidgetsTools (required version >= 6.11.0)
 * Qt6SvgWidgets
 * Qt6Network
 * SWIG
 * OpenMP

-- The following REQUIRED packages have been found:

 * FFmpeg
 * PkgConfig
 * X11
 * OpenCV
 * Threads
 * OpenGL
 * SDL2, low level access to audio, keyboard, mouse, joystick, and graphics hardware, <https://www.libsdl.org/>
 * JNI
 * Lua
 * PerlLibs
 * PHP
 * Python3
 * Ruby
 * TCL
 * Qt6
 * ZLIB

-- The following features have been disabled:

 * Tests
 * Doxygen
 * Clang Format
 * Module: NDI
 * Module: SpatialAudio
 * SWIG: C#
 * SWIG: Node.js

-- The following OPTIONAL packages have not been found:

 * ClangFormat (required version == 14)

-- Configuring done (5.4s)
CMake Error at src/modules/glaxnimate/glaxnimate/src/core/CMakeLists.txt:144 (target_link_libraries):
  Target "GlaxnimateCoreBase" links to:

    Qt5::Core

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



-- Generating done (0.1s)
CMake Warning:
  Manually-specified variables were not used by the project:

    SYSCONF_INSTALL_DIR


CMake Generate step failed.  Build files cannot be regenerated correctly.

```

**Expected behavior**
A clear and concise description of what you expected to happen.



**Desktop (please complete the following information):**
 - OS: [e.g. iOS]                   linux: aosc os   (https://aosc.io/)
 - OS Version [e.g. 22]            13.1.6
- MLT/melt version:          v7.38.0

**Additional context**
Add any other context about the problem here.

--- a/src/modules/glaxnimate/CMakeLists.txt
+++ b/src/modules/glaxnimate/CMakeLists.txt
@@ -26,6 +26,10 @@ function(mlt_add_glaxnimate_module ARG_TARGET)
 
 
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+    set(QT_MAJOR_VERSION ${ARG_QT_VERSION})
+    set(QT_VERSION_MAJOR ${ARG_QT_VERSION})
+
     add_subdirectory(glaxnimate/src/core/)
 
     add_library(${ARG_TARGET} MODULE producer_glaxnimate.cpp)

